daisy/workflow.go

```diff
@@ -89,7 +89,9 @@ func (v *Var) UnmarshalJSON(b []byte) error {
 // Workflow is a single Daisy workflow workflow.
 type Workflow struct {
 	// Populated on New() construction.
-	Cancel chan struct{} `json:"-"`
+	Cancel       chan struct{} `json:"-"`
```

Comment:
Can this be non-exported to avoid direct use? It would be a breaking change but using previous package version would work.

Reply:
+1, I'm fine with making this private

Reply:
I found some legitimate usecases for reading the channel, and given that it'd be a backward-incompatible change, I'm hesitant overall. That said, I overall agree that exposing the channel has downsides.

I'd propose punting this to its own PR if we feel that it's a good idea?

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/1521d107ad566e886a627344a8b1055165de7a9b/cli_tools/common/utils/daisy/daisy_utils.go#L387